public/components/create_report/delivery/__tests__/__snapshots__/delivery.test.tsx.snap

```diff
@@ -0,0 +1,136 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<ReportDelivery /> panel render component 1`] = `
+<div
+  class="euiPanel euiPanel--paddingLarge euiPageContent"
+>
```

Comment:
btw, not a fan to create snapshot for large component. What I mean, not necessary to create snapshot for all component. let's say if what we do is just run snapshot and update snapshot file. it doesn't help too much to ensure quality.

I would only add snapshot to small component, or reusable component. this could save us time.

some good reading below.

https://joshdover.com/when-not-to-use-jest-snapshots/

Reply:
Thanks, will look- plan to add Cypress test suite as well